src/main/java/TasksJavaCore3/Task2.java

```diff
@@ -0,0 +1,41 @@
+package TasksJavaCore3;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Task2 {
+
+    //Task 2.4
+    public static Integer[] getEvenNumbersArray() {
+        List<Integer> list = new ArrayList<>();
+        for (int i = 1; i < 10; i++) {
+            list.add(i);
+        }
+        return list.stream()
+                .filter(number -> number % 2 == 0)
+                .toArray(Integer[]::new);
+    }
+
+    //Task 2.5
+    public static List<String> filterAndSortStrings(List<String> inputList){
+        return inputList.stream()
+                .filter(filteredValues -> filteredValues.startsWith("c"))
+                .map(String::toUpperCase)
+                .sorted((sortedInDescOrder1, sortedInDescOrder2) -> sortedInDescOrder2.compareTo(sortedInDescOrder1))
```

Comment:
you can simplify the sorting by using Comparator.reverseOrder() instead of a lambda expression, this makes the intention more readable